apps/language_server/lib/language_server/server.ex

```diff
@@ -226,11 +226,12 @@ defmodule ElixirLS.LanguageServer.Server do
     end
   end
 
-  defp handle_notification(did_change_configuration(settings), state) do
-    settings = Map.get(settings, "elixirLS", %{})
+  defp handle_notification(did_change_configuration(%{"elixirLS" => settings}), state) do
```

Comment:
@akash-akya thought this little extra match it's worth it, to make it super safe 👍🏼

`defp handle_notification(did_change_configuration(%{"elixirLS" => settings = %{}}), state) do`